documentation/getting_started/configuring-addresses.adoc

```diff
@@ -8,7 +8,7 @@ messages. Currently, {ProductName} supports 4 different address types:
 * Direct anycast addresses
 * Direct broadcast addresses
 
-See the <<address_model>> for details.
+See the xref:con-address-space-messaging[address space] for details.
```

Comment:
For more details, see xref:con-address-space-messaging[address space].

Reply:
Fixed